pandas/io/formats/format.py

```diff
@@ -1667,7 +1667,7 @@ def _format_strings(self) -> list[str]:
 
 
 def format_percentiles(
-    percentiles: (np.ndarray | list[int | float] | list[float] | list[str | float]),
+    percentiles: (np.ndarray | Sequence[float]),
```

Comment:
Was there a reason why the function was supposed to accept strings? @jreback I don't think it supports str as one of the first lines is`np.isclose(percentiles.astype(int), percentiles)` which should fail for str (didn't try it)?

Reply:
don't remember. happy to eliminate it (assume we don't have tests / usage for this)